comfy/ldm/models/autoencoder.py

```diff
@@ -52,7 +54,7 @@ def __init__(
 
         if self.use_ema:
             self.model_ema = LitEma(self, decay=ema_decay)
-            logpy.info(f"Keeping EMAs of {len(list(self.model_ema.buffers()))}.")
```

Comment:
Replacing `logpy` with python's built-in logging module. `logpy` is not used in this project.